package.json

```diff
@@ -48,10 +48,15 @@
       "default": false,
       "description": "Don't show items matched by the `Ignored Names` core config setting."
     },
-    "sortFoldersBeforeFiles": {
-      "type": "boolean",
-      "default": true,
-      "description": "When listing directory items, list subdirectories before listing files."
+    "sortFolders": {
+      "type": "string",
+      "default": "Before Files",
+      "description": "When listing directory items, where subdirectories will be listed.",
+      "enum": [
+        "Before Files",
+        "After Files",
+        "Alongside Files"
+      ]
```

Comment:
This definition should be changed to something along the lines of
``` json
"enum": [
  { "value": "before", "description": "Before Files" },
  { "value": "after", "description": "After Files" },
  { "value": "alongside", "description": "Alongside Files" }
]
```
This will obviously also mean changing the current string comparisons throughout this PR.